tdrs-backend/tdpservice/parsers/schema_defs/tanf/t6.py

```diff
@@ -276,7 +276,10 @@
             startIndex=2,
             endIndex=7,
             required=True,
-            validators=[],
+            validators=[
```

Comment:
Eric and I think these validators were not included to suppress multiple versions of the same error from appearing in the error report, but chose to add them back for consistency across the schemas.